wallet-client/CLI.hs

```diff
@@ -8,21 +8,27 @@ module CLI
   , fromProtocol
   ) where
 
+import           Data.Monoid (Last)
+
 import           Options.Applicative
 
 import           Ouroboros.Consensus.Node.ProtocolInfo
 import           Ouroboros.Consensus.NodeId (CoreNodeId (..))
 
 import           Cardano.Node.CLI
+import           Cardano.Shell.Constants.CLI
+import           Cardano.Shell.Constants.PartialTypes (PartialGenesis (..))
 
 data CLI = CLI {
     cliCoreNodeId   :: CoreNodeId,
     cliNumCoreNodes :: NumCoreNodes,
-    cliProtocol     :: Protocol
+    cliProtocol     :: Protocol,
+    cliGenesis      :: (Last PartialGenesis)
```

Comment:
These parens are redundant.

Reply:
And again, that was already done in a recent commit.